File: src/main/java/gg/mineral/practice/commands/settings/DayCommand.java
package gg.mineral.practice.commands.settings;

import org.bukkit.entity.Player;

import gg.mineral.practice.commands.PlayerCommand;
import gg.mineral.practice.managers.ProfileManager;
import gg.mineral.practice.util.messages.impl.ChatMessages;

public class DayCommand extends PlayerCommand {

    public DayCommand() {
        super("day");
    }

    @Override
    public void execute(Player player, String[] args) {
        ChatMessages.TIME_SET_DAY.send(player);
        player.setPlayerTime(0L, false);
        ProfileManager.getOrCreateProfile(player).setNightMode(false);
    }
}

File: src/main/kotlin/gg/mineral/practice/commands/settings/DayCommand.kt
package gg.mineral.practice.commands.settings

import dev.rollczi.litecommands.annotations.command.Command
import dev.rollczi.litecommands.annotations.context.Context
import dev.rollczi.litecommands.annotations.execute.Execute
import gg.mineral.practice.entity.appender.CommandSenderAppender
import gg.mineral.practice.util.messages.impl.ChatMessages
import org.bukkit.entity.Player

@Command(name = "day")
class DayCommand : CommandSenderAppender {
    @Execute
    fun execute(@Context player: Player) {
        player.send(ChatMessages.TIME_SET_DAY)
        player.setPlayerTime(0L, false)
    }
}